src/analysis/explorerHomeView.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import {CredGraph} from "../core/credrank/credGraph";
+import {type TimestampMs} from "../util/timestamp";
+import {type Interval} from "../core/interval";
+import * as Grain from "../core/ledger/grain";
+
+export type DataOverTime = $ReadOnlyArray<number>;
+
+export type UserRow = {|
+  +name: string,
+  +credEarnedInTimeRange: number,
+  +grainHarvestedInTimeRange: Grain.Grain,
+  +grainBalance: Grain.Grain,
+  +allTimeCred: DataOverTime,
+  +category: UserCategory,
+|};
+
+export type UserCategory = "participant" | "bot" | "project" | "organization";
+
+export type ExplorerHomeView = {|
```

Comment:
Kind of a nit, but can we rename this to something like `ExplorerHomeData`? I think naming it `...View` invites some potential confusion with the Model, View, Controller design pattern (where a View is responsible for rendering in a UI). So when I saw the name `View` I was expecting it to be a class of some kind, not a data type.

Reply:
yep